app/assets/stylesheets/koi/components/_index-table.scss

```diff
@@ -62,15 +62,30 @@ $row-height: 48px !default;
       max-height: 3rem;
       padding: 0;
     }
+  }
 
-    &.number {
+  td {
+    &.katalyst--tables-column-currency,
+    &.katalyst--tables-column-number {
       text-align: right;
       padding-right: 0.5rem;
     }
   }
 
   th {
     font-weight: bold;
+
+    &.katalyst--tables-column-s {
+      width: 5rem;
+    }
+
+    &.katalyst--tables-column-m {
+      width: 10rem;
```

Comment:
This value is too big for dates and to small for date times. How did you pick these values?

Reply:
This was a general assumption we had. Updated to use 8rem for dates and 12rem for datetimes.

```css
&.koi--tables-col-xs {
  width: 5rem;
}

&.koi--tables-col-s {
  width: 8rem;
}

&.koi--tables-col-m {
  width: 12rem;
}

&.koi--tables-col-l {
  width: 20rem;
}
```